service/src/test/java/com.codeforcommunity/processor/BlockInfoProcessorImplTest.java

```diff
@@ -1,313 +0,0 @@
-package com.codeforcommunity.processor;
```

Comment:
Why was this file deleted? This looks like it should be included.

Reply:
Not sure how this got deleted. Will add back.

Reply:
Looks like it's been added back. Thanks.